Sunset notebooks (#5644)
JetBrains/kotlin-web-site: docs/topics/data-analysis/data-analysis-connect-to-db.md

   %use dataframe
   ```

   Run the code cell with the `%use dataframe` line before any other code cells
   to make sure the DataFrame library and its APIs are available in the notebook.

> To follow the tutorial, you can also use DataFrame as an ordinary [Gradle](https://kotlin.github.io/dataframe/setupgradle.html) or [Maven](https://kotlin.github.io/dataframe/setupmaven.html) dependency.

Reviewer: ```suggestion
To follow the tutorial, you can also use DataFrame as a [Gradle](https://kotlin.github.io/dataframe/setupgradle.html) or [Maven](https://kotlin.github.io/dataframe/setupmaven.html) dependency.
```

Reviewer: Unnecessary
